packages/ajax/src/Cache.js

```diff
@@ -3,47 +3,94 @@ import './typedef.js';
 export default class Cache {
   constructor() {
     /**
-     * @type {{ [requestId: string]: { createdAt: number, response: CacheResponse } }}
+     * @type CachedRequests
      * @private
      */
     this._cachedRequests = {};
+
+    /**
+     * @type {number}
+     * @private
+     */
+    this._size = 0;
+  }
+
+  /**
+   * The size of the cache
+   * @returns {number}
+   */
+  get size() {
```

Comment:
I see with the updated implementation size is no longer have to be part of the public API of Cache class, so  better to keep it "private" in JS context

Reply:
Ok, good point! It is done.